dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/FileTool.java

```diff
@@ -56,6 +75,18 @@ public String getURI(FileAsset file, long languageId){
 			return "";
 		}
 	}
+
+	public String getURI(final Contentlet contentlet, long languageId){
```

Comment:
You can avoid last ugly else doing such as
```
String uri = StringPool.BLANK;


if (contentlet instanceof FileAsset) {
			uri = getURI((FileAsset) contentlet, languageId);
		} else if (contentlet.isDotAsset()){
			uri = UtilMethods.espaceForVelocity(
					String.format("dA/%s", contentlet.getIdentifier())
			);
		}
return uri;
```

Reply:
done https://github.com/dotCMS/core/pull/20116/commits/c2de26da737fbd57ed7d67a22425f7cd16a99955